packages/openapi-generator/src/file-serializer/schema.spec.ts

```diff
@@ -108,13 +108,35 @@ describe('serializeSchema for object schemas', () => {
 });
 
 describe('serializeSchema for array schemas', () => {
+  it('serializes array schema', () => {
+    expect(
+      serializeSchema({
+        uniqueItems: false,
+        items: { type: 'string' }
+      })
+    ).toEqual('string[]');
+  });
+
   it('serializes array schema with nested array', () => {
     expect(
       serializeSchema({
         uniqueItems: false,
-        items: { items: { type: 'string' } }
```

Comment:
[req] Please don't remove this test as it's still a valid case

Reply:
Also, I meant to add a test with an example along the lines of what the user has, i.e., an object with array items:

```
 serializeSchema({
        properties: [
          {
            name: 'objectArrayProp',
            required: true,
            schema: {
              uniqueItems: false,
              items: {
                properties: [
                  {
                    name: 'name',
                    required: true,
                    schema: { type: 'string' },
                    schemaProperties: {}
                  }
                ],
                additionalProperties: { type: 'any' }
              }
             },
            schemaProperties: {}
          }
        ],
        additionalProperties: { type: 'any' }
      })
    )
```
This test can go in the object schemas section

Reply:
I see. I didn't think we needed to match the exact case as the user ticket since we already have a test case for serializing object schema with properties. I have added it as requested.